docs/en/student-examples.md

```diff
@@ -0,0 +1,134 @@
+# Student Help Page
+
+This page describes how to interact with Ypovoli as a user.
+
+
+## Login/Logout
+
+**Login**
+
+- On the login page, there is a button with the text "Ugent login".
+
+![login button](../assets/login-button.png)
+- Click on this button. You will be redirected to the Ugent login page.
+- Select the desired account and follow the login steps. You will be redirected to the dashboard.
+
+**Logout**
+
+- Click on your name in the navigation bar.
+
+![logout button](../assets/en/logout-button.png)
+
+## Change Language
+
+- Click on the arrow next to EN at the top of the navigation bar and select the desired language.
+
+![language choice button](../assets/en/lang-change.png)
+
+## Change Mode
+
+::: danger Caution!
+- Only possible if the user has multiple roles like student, teacher, assistant, or admin.
+:::
+
+- Click on the arrow next to the currently selected role in the navigation bar at the top.
+- Choose the new role with which you want to view the page.
+
+![mode choice button](../assets/modus-change.png)
+
+## Enroll/leave in Course
+
+<div style="display: flex; align-items: center;">
+    <img src="../assets/en/join-course.png" alt="course enroll button" style="width: auto; height: auto; margin-right: 10px;">
+    <img src="../assets/en/leave-course.png" alt="course leave button" style="width: auto; height: auto;">
+</div>
+
+**Enroll**
+
+- Click on courses in the navigation bar.
+- Find the correct course using the filters on the left side of the page.
+- Click on the enroll button next to the correct course.
+- You are successfully enrolled in this course.
+
+**leave**
+- Option 1:
+  - Click on courses in the navigation bar.
+  - Find the correct course using the filters on the left side of the page.
+  - Click on the unenroll button next to the correct course.
+  - You are successfully leave from this course.
+- Option 2:
+  - Go to the detail page.
+  - Click on the unenroll button at the bottom.
+  - A popup appears.
+  - Confirm by clicking the leave button or cancel with the cancel button.
+
+## Course Details
+- Click on Dashboard in the navigation bar.
+- Scroll down to the "My Courses" section.
+- Click on the details button next to the chosen course.
+
+![course card](../assets/course-card.png)
+
+::: tip
+- If you want to view a course from a previous academic year, select the correct year in the academic year selector.
+:::
+
+## View Projects
+
+![projects list](../assets/en/project-list.png)
+
+- Option 1:
+  - Click on Dashboard in the navigation bar.
+  - At the top of the page is a section "Current Projects".
```

Comment:
At the top of the page !!there!!  is a section "Current Projects".

Wat do you think?